apis/server.d.ts

```diff
@@ -6,7 +6,9 @@ import { Application, RequestHandler } from 'express'
 import { XOR } from './internal/util'
 
 type _cds = typeof cds
-type cds_services = { [name: string]: Service }
+type service_dict = { [name: string]: Service };
+// interface instead of type so users can insert their actual Services via module augmentation
+interface cds_services extends service_dict {}
```

Comment:
```suggestion
// interface instead of type so users can insert their actual Services via module augmentation
interface cds_services {
  [name: string]: Service
}
```

Sorry, I was probably a bit vague. 🙂 What I was asking is: why don't we just fully embrace the `interface`-way here, or is there a specific reason why you want to also keep the synthetic `service_dict`-type?

Reply:
Ah, I see. I kinda forgot about index signatures, haha.